src/components/App.js

```diff
@@ -45,7 +50,7 @@ const App = () => {
       </Container>
       <TodoList>
         {todos.map((todo) => (
-          <Todo key={todo.id} id={todo.id}>
+          <Todo key={todo.id} id={todo.id} icon={<BsTrash onClick={() => deleteToDo(todo.id)} />}>
```

Comment:
Si bien está bueno que llegado el caso el ícono de borrado pueda ser configurable, dado nuestra aplicación es probable que no necesitemos tanta personalización, ya que va a ser el mismo ícono en todos los todos. Por lo tanto, tenerlo como prop lo vuelve más complejo que si lo renderiza directamente. En vez del ícono, debería tener una prop para pasarle la función que se va a ejecutar cuando se borra el todo.

```suggestion
          <Todo key={todo.id} id={todo.id} onDelete={deleteToDo}>
```

Reply:
Consulta, en este caso estas asignandole la funcion onDelete al Todo, pero esta funcion corresponde al evento de click en el icono, estaría bien de esa forma? Gracias

Reply:
Sí, la idea es que el ícono esté dentro del componente Todo, ese prop así solo no hace nada, hay que asignarlo luego al ícono.